### PR TITLE
Create buddha-and-metaphysics_wijesekera.md

### PR DESCRIPTION
**Pull request by JJM317**
Can we make an upanishad tag? Also, I noticed that Motilal Banarsidass doesn't have a slug. Did I miss it? If it doesn't, I think it is a large enough publisher to warrant its own slug, no?

### CODE REVIEW

File: _content/papers/buddha-and-metaphysics_wijesekera.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddha and Metaphysics"
+authors:
+  - wijesekera
+editor: "M. H. F. Jayasuriya"
+external_url: "https://archive.org/details/in.ernet.dli.2015.463583/page/n3/mode/2up"
```
Comment (buddhist-uni) [inline _content/papers/buddha-and-metaphysics_wijesekera.md:6]: This cut of the PDF is unique to OBU, so I uploaded it to our internal storage. Link to it like so instead:

```yaml
file_links:
  - "exclusive_01/The%20Buddha%20and%20Metaphysics%20-%20Wijesekera.pdf"
```

File: _content/papers/buddha-and-metaphysics_wijesekera.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/19rYXVNrPkyrvmdrSTovUrXR6-17Sbdp4/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - metaphysics
+  - indic-religions
+  - suttas
+  - upanishads
+year: 1994
```
Comment (buddhist-uni) [inline _content/papers/buddha-and-metaphysics_wijesekera.md:16]: Originally published 1941, so let's use that year

File: _content/papers/buddha-and-metaphysics_wijesekera.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+oclc: 30546679
```
Comment (buddhist-uni) [inline _content/papers/buddha-and-metaphysics_wijesekera.md:17]: This oclc belongs on [the monograph](https://github.com/buddhist-uni/buddhist-uni.github.io/blob/main/_content/monographs/buddhist-and-vedic-studies_wijesekera.md?plain=1) not the chapter.

File: _content/papers/buddha-and-metaphysics_wijesekera.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+publisher: Motilal Banarsidass
```
Comment (buddhist-uni) [inline _content/papers/buddha-and-metaphysics_wijesekera.md:18]: In quotes for now. I'll add the publisher as a linked thing in a future change, good idea! 😊

File: _content/papers/buddha-and-metaphysics_wijesekera.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+editor: "M. H. F. Jayasuriya"
+external_url: "https://archive.org/details/in.ernet.dli.2015.463583/page/n3/mode/2up"
+drive_links:
+  - "https://drive.google.com/file/d/19rYXVNrPkyrvmdrSTovUrXR6-17Sbdp4/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - metaphysics
+  - indic-religions
+  - suttas
```
Comment (buddhist-uni) [inline _content/papers/buddha-and-metaphysics_wijesekera.md:14]: Unnecessary to tag as `suttas` I think.

File: _content/papers/buddha-and-metaphysics_wijesekera.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddha and Metaphysics"
+authors:
+  - wijesekera
+editor: "M. H. F. Jayasuriya"
+external_url: "https://archive.org/details/in.ernet.dli.2015.463583/page/n3/mode/2up"
+drive_links:
+  - "https://drive.google.com/file/d/19rYXVNrPkyrvmdrSTovUrXR6-17Sbdp4/view?usp=drive_link"
+formats: [pdf]
+course: view
```
Comment (buddhist-uni) [inline _content/papers/buddha-and-metaphysics_wijesekera.md:10]: Hmmm  Rereading this, I'm wondering if it belongs here or shouldn't "fall down" to `cosmology` 🤔  What do you think?

**Comment by JJM317** [comment]
Ya, it's a good question. It would work in either place. I'm leaning
towards keeping it in view rather than moving it to cosmology, though. It
feels like this paper is about the views underlying different cosmologies
and why the Buddha thought some of those views are incorrect (eternalists
and nihilists).
 Just my two cents
~Joe
--------------------

**Comment by buddhist-uni** [comment]
> different cosmologies... eternalists
and nihilists

Yeah, if that's the best case that can be made, then we should just move it to `cosmology` 😆

**Comment by buddhist-uni** [comment]
Oops! One minor mistake here: this should have been in "excerpts" rather than "papers" as it's from a collection by one author 😁

**Comment by buddhist-uni** [comment]
Okay. Publisher added: https://buddhistuniversity.net/publishers/mlbd 😊

**Comment by JJM317** [comment]
Any chance of rethinking this category? In the publishing and library
worlds, excerpt doesn't mean a whole paper from an edited volume. It's
simply a short extract of a work, no? Actually, 'book chapter' could be a
category, too, for any chapter of a monograph. Monographs and edited
volumes are different items, agreed?

--------------------

**Comment by buddhist-uni** [comment]
Hmmm, yeah, the schema was based on the Bibtex format. "Excerpt" was my term for their `@inCollection` type. We can discuss more on Wednesday, but yeah, this is baked into the URLs at this point, so it would be pretty time consuming to change now...

**Pull request merged**